app/components/Button/index.js

```diff
@@ -7,37 +7,64 @@
  */
 
 import React, { PropTypes, Children } from 'react';
+import styled from 'styled-components';
 
-import styles from './styles.css';
+const buttonStyling = `
+  display: inline-block;
+  box-sizing: border-box;
+  padding: 0.25em 2em;
+  margin: 0;
+  text-decoration: none;
+  border-radius: 4px;
+  -webkit-font-smoothing: antialiased;
+  -webkit-touch-callout: none;
+  user-select: none;
+  cursor: pointer;
+  outline: 0;
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-weight: bold;
+  font-size: 16px;
+  border: 2px solid #41ADDD;
+  color: #41ADDD;
 
-function Button(props) {
-  const className = props.className ? props.className : styles.button;
+  &:active {
+    background: #41ADDD;
+    color: #FFF;
+  }
+`;
+export const AnchorButton = styled.a`${buttonStyling}`;
```

Comment:
Your totally right, it could be called `A` or `Link` and that makes sense. I simply choose `AnchorButton` because I thought it was more descriptive.